docs/user-guide/overview.md

```diff
@@ -10,10 +10,23 @@
 >In the tenant menu you will have access to all the tenants related to the user and you can also quickly switch between one tenant and another.
 ><div align=center><img width="800" src="/assets/images/top-menu/tenant-menu.jpg"/></div>
 
+## Tenants
+>A tenant is like a sales channel. Let's say you have a bussines A where you sale some kind of products so you are going to have a tenant with some connected integrations to manage your products/orders etc. Now, let's say you have another bussines B. For that you can create another tenant and create there connected integrations to manage others product/orders. 
```

Comment:
"bussines **A** where you" to "**business** where you" check the working in google doc.